app/components/Modals/VotingModal/VotingModal.jsx

```diff
@@ -0,0 +1,246 @@
+// @flow
+import React, { Component } from 'react'
+
+import BaseModal from '../BaseModal'
+import Button from '../../Button'
+import styles from './styles.scss'
+import classNames from 'classnames'
+
+type Props = {
+  hideModal: Function,
+  showSuccessNotification: Function
+}
+
+type State = {
+  infoShowing: boolean,
+  confirmationShowing: boolean,
+  votes: Object
+}
+
+const textStrings = {
```

Comment:
I partially understand why you chose to do this, but I think its much better if you put those strings in JSX elements. We sure can use `i18n` in v2, but this isn't needed here.